docs/pages/slimbanner.js

```diff
@@ -17,9 +17,18 @@ export default function SlimBannerPage({ generatedDocGen }: {| generatedDocGen:
         description={generatedDocGen?.description}
         defaultCode={`
 <SlimBanner
-  type="info"
-  message="Idea Pins are now available across platforms."
+  dismissButton={{
+    accessibilityLabel: 'Dismiss',
+    onDismiss: () => {},
+  }}
   iconAccessibilityLabel="Information"
+  message="Idea Pins are now available across platforms."
+  primaryAction={{
+    accessibilityLabel: 'Apply',
```

Comment:
This should probably be "Apply for Idea Pin access" or something similar (unclear what exactly Apply means here)